docs/ОтладкаСценариев.md

```
@@ -0,0 +1,54 @@
+
+# Отладка сценариев (тесты)
+
+Эффективно разрабатывать правила-сценарии обработок сразу вместе с тестом.
+
+1. Установить 1testrunner для возможности модульного тестирования сценариев обработки
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)